css/main.css

```diff
@@ -294,106 +312,162 @@ body {
   background-color: #fafafa;
 }
 .head_section2_features {
-  margin-top: 64px;
+  margin: 0;
+  padding-top: 64px;
   font-size: 36px;
   line-height: 44px;
+  font-weight: 600;
 }
 .head_section_technology {
+  margin: 0;
+  font-weight: 600;
   font-size: 36px;
   line-height: 44px;
   padding-top: 64px;
 }
 .features_blocks {
-  margin-top: 64px;
+  padding-top: 64px;
+  padding-bottom: 124px;
 }
 .elements_for_features_technology {
   list-style-type: none;
 }
 .features_and_technology_list_item {
   display: inline-block;
 }
+.img_features {
+  max-width: 100%;
+}
 .features_block_text_on_image {
-  width: 293px;
-  height: 198px;
+  width: 83%;
+  min-height: 202px;
   background: #fff;
   box-shadow: 0px 5px 30px rgba(0, 0, 0, 0.1);
   border-radius: 8px;
-  margin-bottom: 124px;
-  margin-top: -120px;
-  margin-left: 28px;
-  margin-right: 28px;
+  margin-top: -96px;
+  margin-left: 9%;
   position: relative;
 }
+.head_of_features_block {
+  margin: 0;
+  padding-left: 32px;
+  padding-right: 32px;
+  padding-top: 32px;
+  font-size: 18px;
+  line-height: 22px;
+  height: 40px;
+  color: #101010;
+}
 .features_text {
+  margin: 0;
+  height: 57px;
```

Comment:
r u sure?

Reply:
"height: 57px;" helped me to position content in the <p> tag like in our model.
Without it we haven't approximate values.
For example, in the first PrtSc we see content position like in our model.
In the second photo we have reverse situation.
![2](https://user-images.githubusercontent.com/74420307/102138533-2c40df00-3e65-11eb-8fdc-821dd94c4e77.png)
![1](https://user-images.githubusercontent.com/74420307/102138541-2d720c00-3e65-11eb-8fba-211fa6cd6c04.png)

We could delete height and add smaller padding bottom, but we haven't effect like in our model.